app/modules/site_settings/helpers.py

```diff
@@ -476,10 +476,14 @@ def is_valid_value(cls, cf_defn, value):
                 value[0] = float(value[0])
             if isinstance(value[1], int):
                 value[1] = float(value[1])
-            if not util.is_valid_latitude(value[0]):
+            if not value[0] and not value[1]:
```

Comment:
am i making this up, or does `not 0.0` evaluate to `True` in python?  if so, i think these may need to be made into `value[X] is not None`.   and i guess probably a good reason for us to add a test to confirm `[0, 0]` indeed passes.